README.markdown

```diff
@@ -5,10 +5,12 @@ Puppet Dashboard is a web interface for [Puppet](http://www.puppetlabs.com/).
 It can view and analyze Puppet reports, assign Puppet classes and parameters to
 nodes, and view inventory data and backed-up file contents.
 
+[![Build Status](https://travis-ci.org/sodabrew/puppet-dashboard.svg?branch=master)](https://travis-ci.org/sodabrew/puppet-dashboard)
+
 Dependencies
 ------------
 
-* Ruby 2.2, 2.3, 2.4, 2.5
+* Ruby 2.3, 2.4, 2.5
```

Comment:
I'd like to keep 'official' ruby 2.2 support for the dashboard 3.0 release so people who are still on ruby 2.2 can update to a non-ancient 'released' version of dashboard.  or is nobody on ruby 2.2 anymore?

Reply:
Nah, kill it. Nobody is on Ruby 2.2 anymore. It's already out of service window and was not shipped as the system Ruby in any major releases.